client/app/Components/Features/Session/Users/Components/UserView.tsx

```diff
@@ -1,12 +1,17 @@
 import { FunctionComponent, useState } from "react";
 import { UserCard } from "./UserCard";
 import { useSessionUserStore } from "../Stores/UserStore";
+import { useTaskStore } from "../../Tasks/Stores/TaskStore";
+import { Status } from "../../../../../Types/Status";
 
 export const UserView: FunctionComponent<{}> = () => {
   const [showImagesOnly, setShowImagesOnly] = useState<boolean>(true);
 
   const { users } = useSessionUserStore();
 
+  const { tasks, userAlreadyVoted } = useTaskStore();
+  const task = tasks.find((item) => item.status == Status.Open);
```

Comment:
I think you can just use the findOpenTask function here that you added to the TaskStore